docs-site/content/0.19.0/guide/features/scoped-API-keys.md

```diff
@@ -0,0 +1,47 @@
+# Scoped API Keys
+
+API keys are at the core of Typesense. To perform any action with Typesense, you need API keys. Typesense also allows access control on API keys. You can define capabilities as to what a user can or cannot do. You can also restrict access to a specific document or collection. In the case of a multi-tenant environment, you can scope API keys to a particular subset. This is helpful when you have indexed data from multiple tenants in your Typesense server and want to restrict users to only access their subset of data.
+
+Typesense allows you to create API keys that have pre-defined filters embedded in them. So, whenever you run a search query with these API keys, those filters are automatically applied and cannot be overridden. You can then provide those search API keys to users and they would only be able to access the data that is allowed by the set filter. To create scoped API keys, you just need a parent key. 
+
+Let's create a scoped search API key that will restrict users to only access documents that have the company id 124:
```

Comment:
Could you add a sample set of documents, showing company_id set to different values. And how a user using a scoped key with company_id: 124 only gets records with that company_id and not others?